src/componnets/NavBar.js

```diff
@@ -0,0 +1,46 @@
+import { NavLink } from "react-router-dom";
+import classes from "./NavBar.module.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCartShopping } from "@fortawesome/free-solid-svg-icons";
+
+const NavBar = ({ productsOnCart }) => {
+  const totalProducts = productsOnCart.reduce((current, total) => {
+    return parseInt(current) + parseInt(total.amount);
```

Comment:
I think we know `current` and `total.amount` will always be numbers so we wouldn't need to parse from a string.

Reply:
Thanks